src/Components/PurchaseCard/PurchaseCard.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  Grid,
+  GridItem,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+
+const PurchaseCard = () => {
+  return (
+    <>
+      <Card>
+        <CardBody>
+          <Grid>
+            <GridItem
+              md={6}
+              lg={8}
+              component={TextContent}
+              className="pf-u-pr-2xl"
+            >
+              <Text component={TextVariants.h2} className="pf-u-mb-md">
+                Continue your experience with Red Hat OpenShift service on AWS
+              </Text>
+              <Text
+                component={TextVariants.p}
+                className="pf-u-color-200 pf-u-mb-lg"
+              >
+                Get started with your own dedicated cluster on AWS, managed by
+                Red Hat and Amazon, and even move your application from Dev
+                Sandbox.
+              </Text>
+              <Flex className="pf-u-flex-direction-column pf-u-flex-direction-row-on-md pf-u-mt-lg pf-u-mt-0-on-md">
+                <Button
+                  variant="secondary"
+                  component="a"
+                  href="https://console.redhat.com/beta/openshift/create/rosa/welcome"
```

Comment:
You shouldn't use direct `hrefs` as that'll perform full page refresh. Instead you should import `useChrome` and from it use `push`
```JS
const push = useChrome(({ chromeHistory: { push } = {} }) => push);
```

And over here add `onClick`
```suggestion
                  href="./openshift/create/rosa/welcome"
                  onClick={(e) => {
                    e.preventDefault();
                    push('./openshift/create/rosa/welcome'); // the dot is here required since we want to respect if we're on beta or not
                  }}
```